viz.yaml

```diff
@@ -101,7 +101,7 @@ parameter:
   -
     id: display_crs	
     comment: projecting into
-    crs_str: "+init=epsg:5070"
+    crs_str: "+init=epsg:4326" #needed to play nice w/ ggmap?
```

Comment:
I need to look into what ggmap does. This is screwed up. Apparently it does the projection for you from lat/lon inputs to EPSG:3857 which is what the underlying tiles come to us in.

Reply:
Based on your comment below ("for this, I think we are doing the right thing"), should we leave this as-is? Thanks for digging up that blog post btw!

Reply:
Yeah. Leave it. Unless you want to leave ggmap behind.

Usually, you pass data to a map in the projection of the map and, because the tiles are projected in [web mercator (EPSG:3875)](https://en.wikipedia.org/wiki/Web_Mercator), theoretically, all data on a map that uses that projection should have to be in that projection. 

Somehow, ggmap works around that normal way of doing maps. You just pass it lat/lon stuff (EPSG:4326) and it figures out how to project it into web mercator or it somehow reprojects the tiles into some other projection, I really don't know which way it goes.

Side Note: EPSG:5070 that's listed above is a typical visualization projection for things like watersheds because it preserves area well across most of the US.

Reply:
I think for this particular application, we will keep ggmap but now we can understand it's limitations for the future. 

Right, I remember you talking about EPSG:5070 before but noted that it wasn't working correctly with ggmap. 

THANKS!